app/javascript/theme/components/modals/_login.scss

```diff
@@ -3,6 +3,12 @@
     margin-bottom: 0.75rem;
   }
 
+  &__support {
+    color: $neutral70;
+    margin: 0.5rem 0;
```

Comment:
Let's keep this `margin-top` for the instructional copy.

If you want the extra space in the login modal, let's add an extra `0.5rem` to `margin-top` in the `Login__button` class.

Reply:
I'm not sure I'm following this one. Are you saying remove the margin from the copy and add to the button?